src/config-schema.ts

```diff
@@ -133,6 +133,21 @@ export const configSchema = {
       'Enable or disable stock deduction during the dispensing process. Requires the stock management module to be installed and configured.',
     _default: false,
   },
+  showDiagnosesFromVisit: {
+    _type: Type.Boolean,
+    _description: 'Enable or disable viewing of patient visit diagnoses',
+    _default: false,
+  },
+  showExtraPatientInformationSlot: {
+    _type: Type.Boolean,
+    _description: 'Enable or disable viewing of patient extra information slot diagnoses',
+    _default: false,
+  },
+  showPatientConditions: {
```

Comment:
We also don't need _configuration_ options where things are exposed as extensions. Please remove these.